src/js/environment/environment.ts

```diff
@@ -20,18 +20,20 @@ import {
 import { browserVersion } from './browser-version';
 import { osVersion } from './os-version';
 import { memoize } from 'utils/underscore';
+import { GenericObject } from 'types/generic.type';
 
-const userAgent = navigator.userAgent;
+const userAgent: string = navigator.userAgent;
+const noop: () => void = () => {};
 
-function supportsPassive() {
+function supportsPassive(): boolean {
     let passiveOptionRead = false;
 
     try {
-        const opts = Object.defineProperty({}, 'passive', {
+        const opts: GenericObject = Object.defineProperty({}, 'passive', {
             get: () => (passiveOptionRead = true)
         });
-        window.addEventListener('testPassive', null, opts);
-        window.removeEventListener('testPassive', null, opts);
+        window.addEventListener('testPassive', noop, opts);
+        window.removeEventListener('testPassive', noop, opts);
```

Comment:
Disregard my earlier comment here. Modernizr settled on using `noop` for this test: 
https://github.com/Modernizr/Modernizr/blob/3675d7f6837e01a4237c91f325facf3037c030c5/feature-detects/dom/passiveeventlisteners.js